decentraland/unity-explorer
# Chore: chat command goto extended

## What does this PR change?
Extended chat `/goto` and `/world` commands
- minor terrain related bug-fixes
- removed the need for `.dcl.eth` prefix to jump in worlds
- add `/goto-local x,y` command to be able teleport inside the realm (without being dropped back to  GCity)
- add support to be able to jump to several realms (not only GenesisCity or Worlds), such as
```
"https://sdk-team-cdn.decentraland.org/ipfs/goerli-plaza-main";
"https://sdk-team-cdn.decentraland.org/ipfs/goerli-plaza-main-latest";
"https://sdk-team-cdn.decentraland.org/ipfs/streaming-world-main";
"https://sdk-team-cdn.decentraland.org/ipfs/sdk7-test-scenes-main-latest";
 "https://sdk-test-scenes.decentraland.zone";
```

## How to test the changes?

1. Launch the explorer and try chat commands
2. Jump to world without need of `.dcl.eth` suffix. For example - `/goto yemel`
3. While there, jump inside world coordinates:  `/goto-local 5,5` - that should teleport you inside the world, not teleporting you to GenesisCity
4. Try `/goto sdk` or   `/goto goerli` or  `/goto test` or  `/goto goerli-old`  = this should bring you to respective realm. Keep in mind, since there are many errors in these realms, your experience probably will broken after that, but you should be able to see some scenes from the respective realms.
5. Check that old commands working as before. Try negative coordinates. Try `/goto random` that brings your to random coordinate inside GCity

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
